coops-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -87,7 +94,7 @@ describe('Dashboard - UI', () => {
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     router.push({ name: 'dashboard', query: { filing_id: '123' } })
-    const wrapper = mount(Dashboard, { localVue, store, router, vuetify })
+    const wrapper = shallowMount(Dashboard, { localVue, store, router, vuetify })
```

Comment:
Do all unit tests pass? I thought we needed a deep mount for the sub-components, but maybe not!

Reply:
They do indeed! The child components are stubbed/component placeholders in the shallowmount. Since they are not subject to any real testing here, just to ensure they are present, shallowmount seems to make sense. Shallowmount also helps us test the dashboard in isolation without any conflicts from child components. What are your thoughts on this?

Reply:
Agreed!